src/api.js

```diff
@@ -29,6 +29,12 @@ export const voteOnArticle = (article_id, voteChange) => {
     .patch(`/articles/${article_id}`, { inc_votes: voteChange })
     .then(({ data }) => data);
 };
+
+export const addComment =(article_id, comment) => {
+    return api.post(`/articles/${article_id}/comments`, comment).then(({ data }) => {
+        return data;
+})
+}
```

Comment:
Nice, but instead of sending back the whole data object, is there some more specific we could pass down instead?